### ISSUE TITLE
Allow worker pattern to create FIFO queue

### ISSUE THREAD

**Issue by reyreaud-l** [aws/copilot-cli#2800]
Hey there, thanks for the great feature for worker pattern implemented in https://github.com/aws/copilot-cli/issues/2550.

It would be nice if we could also precise as parameter in the manifest if a FIFO queue is needed.

Something like:
```
subscribe:
  topics:
    - name: jobs
      service: api
      queue:
        fifo: true
```

Thanks!

**Comment by efekarakus** [comment]
Hi @reyreaud-l ! 

Thank you for the feature request 😄 

Funny enough we started exactly with the proposal you have in your description, but had to remove it for the v1.10 launch: https://github.com/aws/copilot-cli/pull/2775

The reason is that to use SQS fifo queues, we need to also provide a `fifo: true` field to the publishers.

```yaml
publish:
  topics:
  - name: mytopic
     fifo: true
```

Once we have SNS fifo enabled we can revert the changes in #2775.

**Comment by efekarakus** [comment]
This feature is now released in [v1.22](https://github.com/aws/copilot-cli/releases/tag/v1.22.0) 🚀 !

For the blog post: https://aws.github.io/copilot-cli/blogs/release-v122/
Release notes: https://github.com/aws/copilot-cli/releases/tag/v1.22.0
